README.md

```diff
@@ -1,8 +1,24 @@
 ![alt tag](https://66.media.tumblr.com/tumblr_lfp90xpDTm1qb9w8so1_250.gif)
 
+# Ping Pong Championship Rules
 
-- New rules must be voted on and have a 2/3 majority in favor of the new rule.
-- If the existing champion is out of the office for 3 days, regardless of reason, excluding weekends, then they give up their title.
-- If the champion challenges someone or accepts a challenge and do not play then they forfiet their title to the opponent.
-- The winner must win 3/5 games.
-- If someone wins 3 consecutive games then they must offer a winner takes all match. The winner of the winner takes all match wins the title.
+## Article 1 - Committee Rulings
+1. Any additional rules must be voted on and win by a 2/3 majority in order to be ratified.
+2. In order to be a voting member, a player must meet *one* of the following requirements:
+    1. Contributing member to the ping-pong-champion-rules git repo.
+    2. Won a championship title at some time in the past.
+    3. A current Vivid Seats employee that has seen the movie [Balls of Fury](http://www.imdb.com/title/tt0424823/)
+
+## Article 2 - Obtaining Championship Status
+1. Championship matches are played best 3 out of 5, 2 serves each to eleven points as per the [ITTF Handbook](http://www.ittf.com/handbook/).
+2. If a player bests their opponent by three consecutive games, they are obligated to offer a 'winner takes all' round. The winner of the 'winner takes all' round becomes the *CHAMPION*.
+3. If during any championship match a player is beaten 11-0, skunk rules go into effect which automatically make the winning player the *CHAMPION*.
+
+## Article 3 - Immunity & Forfeiture
+1. If a player challenges the *CHAMPION*, the *CHAMPION* may do one of three things:
+    1. Accept the challenge and place the title on the line. Backing out of an accepted challenge results in a forfeit.
+    2. Defer the challenge. The current *CHAMPION* may defer a challenge until the next business day. Note that if the *CHAMPION* is out of office (OOO) the next day, the *CHAMPION* retains Championship status. Additionally, it is up to the challenger to re-initiate the challenge at a mutually agreed upon time. If a time cannot be agreed upon or work duties prevent the match from occurring, the *CHAMPION* retains *CHAMPION*ship status.
+    3. Forfeit. Forfeiture immediately bestows the rights and privileges of *CHAMPION*ship status to the challenger.
+2. Any non-*CHAMPION* challenges are called 'Exhibition Matches' and do not bestow any rights or privileges to the participants.
+3. If the existing *CHAMPION* is OOO for 3 working days then they relinquish their Championship status. At this time, any two players can escalate an 'Exhibition Match' to a Championship match.
```

Comment:
I think that if the champ is OOO for 1 working day, Then they loose the title. Its a new idea, but I think its most fair.

Reply:
Committee needs to vote on this... I think it sounds like an easy way for weasels to claim the title for themselves.

Reply:
as the current champion i don't like this 😬

Reply:
even though it would mean i would relinquish my title tomorrow, i would agree that 2 working days be all that is required in order to relinquish the title. 1 seems not long enough, 3 seems like a long time, 2 may be the goldilocks number.

Reply:
I prefer 1 day. What if someone dodges a challenge by working from home? The championship must go on even when they are not in the office. I'm down for a vote.